machine-config/digitalocean.vue

```diff
@@ -190,5 +213,20 @@ export default {
         />
       </div>
     </div>
+    <div class="row mt-20">
+      <div class="col span-6">
+        <ArrayList
+          v-model="tags"
+          :mode="mode"
+          :protip="false"
+          :title="t('cluster.machineConfig.digitalocean.tags.label')"
+          :value-placeholder="t('cluster.machineConfig.digitalocean.tags.placeholder')"
+          :disabled="!isCreate"
+          :add-allowed="isCreate"
+          :remove-allowed="isCreate"
```

Comment:
We don't want to be able to edit tags?

Reply:
My reasoning was:
- When tags were removed from the droplet the original cluster would be deleted and then subsequently replaced with a new one
- In the Ember UI you were only able to add tags to a cluster template, which is not editable after the cluster has been created.

I'll look into a way of removing the tags without the cluster being deleted.

Reply:
If that's that case don't worry about it. I think that's a reasonable choice given that context.